subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/modifiers/InsecureProtocolOption.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.buildinit.plugins.internal.modifiers;
+
+import javax.annotation.Nullable;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum InsecureProtocolOption implements WithIdentifier {
+    WARN("warn"),
+    ALLOW("allow"),
+    UPGRADE("upgrade");
+
+    public static InsecureProtocolOption defaultOption = WARN;
```

Comment:
I think based on some of my other comments elsewhere, we can get rid of this field.

Usually public static constants like this should be in all caps separated by underscores.